### DISCUSSION TITLE
[Frontend] Trilha de estudo para Angular 2+ (TS, JS, HTML e CSS)

### DISCUSSION THREAD

**Discussion by belmirofss** [frontendbr/forum#2090] [Dicas]
## Trilha de estudo para Angular 2+ 

Trabalho com Angular há um tempo e de vez em quando aparecem pessoas pedindo ajuda sobre como aprender Angular, Javascript, HTML, CSS etc. e seu sempre recomendava uma trilha de estudo que montava ali na hora.

Refleti sobre isso e decidi gastar um pouco de energia montando uma trilha de estudo grátis para se aprender Angular.

[Escrevi tudo em um artigo no Medium](https://belmirofss.medium.com/trilha-de-estudo-para-angular-2-28969b82444f) e estou aqui divulgando para caso alguém tenha interesse e ver.

**Comment by Vinicius-Merces** [comment]
Cara, muito obrigado por compartilhar isso, vai me ajudar demais, estou iniciando igual você, através de muito estudo e videos pela internet, e tem uma trilha pra seguir ajuda demais.

**Reply by belmirofss** [reply]
Feliz demais em ler isso. Obrigado!

**Comment by ghost** [comment]


**Reply by belmirofss** [reply]
Obrigado por somar tanto! Não conhecia o freeCodeCamp e curti demais. Editei o texto e adicionei a sugestão.

**Comment by jpferreiradev** [comment]
Cara, muito bom o seu artigo. Vai me ajudar muito no meu estudo, muito obrigado!

**Reply by belmirofss** [reply]
Tamojunto!!
